tests/app/celery/test_tasks.py

```diff
@@ -727,24 +708,20 @@ def test_should_not_send_email_if_team_key_and_recipient_not_in_team(sample_emai
     team_members = [user.email_address for user in sample_email_template_with_placeholders.service.users]
     assert "my_email@my_email.com" not in team_members
 
-    with freeze_time("2016-01-01 11:09:00.00000"):
```

Comment:
We could keep this test as is, remove the parameter and then check the created time matches the freeze time?

Reply:
As discussed in person, we'll leave time out of this test (because no notifications get created in this test, so we can't add in an assertion to check the creation time). In the tests where we are using `save_email` and the email notifications do get created we mostly include an assertion for the creation time, for example [here](https://github.com/alphagov/notifications-api/blob/0c9a4bce595f4e186c6e0fc4d29eb1439eec4103/tests/app/celery/test_tasks.py#L746) and [here](https://github.com/alphagov/notifications-api/blob/0c9a4bce595f4e186c6e0fc4d29eb1439eec4103/tests/app/celery/test_tasks.py#L817)